router/batchrouter/asyncdestinationmanager/bing-ads/offline-conversions/bulk_uploader.go

```diff
@@ -19,15 +19,16 @@ import (
 	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/common"
 )
 
-func NewBingAdsBulkUploader(logger logger.Logger, statsFactory stats.Stats, destName string, service bingads.BulkServiceI, client *Client) *BingAdsBulkUploader {
+func NewBingAdsBulkUploader(logger logger.Logger, statsFactory stats.Stats, destName string, service bingads.BulkServiceI, client *Client, isHashRequired bool) *BingAdsBulkUploader {
```

Comment:
params are increasing so we need do something

Reply:
looks `client *Client` is not used at all

Reply:
instead of isHashRequired, can we pass the destination config and deal with in function

Reply:
I have removed `client` as we are not using that. But I am not sending the entire destination config as of now as only `isHashRequired` is required.